### DISCUSSION TITLE
Problème avec les services de récupération de fichiers (FRW121 - Obtenir un fichier lié à un formulaire Web)

### DISCUSSION THREAD

**Discussion by ChrisThe01** [MTESSDev/FRW#220] [Annonces]
Bonjour,

Dans notre traitement lors de l'appel au service FRW121 - Obtenir un fichier lié à un formulaire Web, nous avons l'erreur suivant:

Appel FRW-Api en erreur - InternalServerError - {"No":"500","Message":"Objet erreur reçu seul","Cible":"/api/v1/Document/Telecharger/MA04GAZ5z2cYOlAjo6M3I0wRYXJq3PurZv026qy9ign89vn7wXhGLKoxjjD0.docx"}

Notre traitement appel votre service en IT.

Merci,

**Comment by LarcM7** [comment]
Bonjour, cela m'a tout l'air d'un cas de formulaire épuré.

Après un certain délai les formulaires et leurs pièces jointes sont épurés.

Selon le numéro séquentiel du formulaire en question il ne datait pas d'aujourd'hui ou d'hier donc ma meilleure hypothèse est qu'il a été épuré. Mais je peux me tromper.

Pouvez-vous recréer un autre formulaire avec une pièce jointe pour vous débloquer? Utiliser un autre cas?

Merci.
